manifests/init.pp

```diff
@@ -1263,4 +1263,18 @@
       fail("Pam is only supported on RedHat, SuSE, Debian and Solaris osfamilies. Your osfamily is identified as <${::osfamily}>.")
     }
   }
+
+  if $ensure_vas == 'present' and ($::osfamily == 'RedHat' or $::osfamily == 'Debian' or ($::osfamily == 'Suse' and $::lsbmajdistrelease != '9')) {
```

Comment:
@ghoneycutt That would lock out people configuring pam to use vas on those platforms until support is added to the vas module though. I'm not sure which is better.